src/pages/CollectionSingleViewPage.tsx

```diff
@@ -448,10 +628,13 @@ const TableTabContent = ({
         const query = params.row.query;
         const urlParams = new URLSearchParams();
         addVariantSelectorToUrlSearchParams(query, urlParams);
+
         const placeString = encodeLocationSelectorToSingleString(locationSelector);
         return params.value ? (
           <Link
-            to={`/explore/${placeString}/AllSamples/Past6M/variants?${urlParams.toString()}`}
+            to={`/explore/${placeString}/AllSamples/${dateRangeUrlFromSelector(
+              dateRangeSelector
```

Comment:
Why do we need the dateRangeSelector here now? Was this a bug before?

Reply:
Yes, the link should open the main page with the date selector specified on the collection page.